.github/workflows/bench.yml

```diff
@@ -117,13 +126,24 @@ jobs:
         with:
           node-version: 20.x
 
+      - name: Create temporary report folder
```

Comment:
I don't get why this is needed

Reply:
I need this folder to download all artifacts and make it easy to delete all unused reports.

There may be unused reports because downloading artifacts will get artifacts from the entire workflow,
so when we run `run all` it will leave many reports unused, and this can be problematic during the job to commit the files.